docker-compose.yml

```diff
@@ -1,21 +1,42 @@
 version: '3.5'
 
 services:
-  web:
-    container_name: hackathonmentors
-    build: .
+  django:
+    container_name: api
+    build: ./api
     command: python hackathonmentors/manage.py runserver 0.0.0.0:8000
     volumes:
-      - .:/build
+      - ./api:/build
     ports:
       - "8000:8000"
     environment:
       DJANGO_SECRET_KEY: /var/run/secrets/django_secret
       EMAIL_SERVICE_NAME: mailchimp
       EMAIL_NEWLETTER_LIST_ID: d574cb485f
+      DATABASE_HOST: db
     secrets:
       - django_secret
 
+  web:
+    container_name: web
+    image: node:15.8.0
+    command: sh entrypoint.sh
+    working_dir: /web
+    volumes:
+      - ./web:/web
+
+  nginx:
+    container_name: nginx
+    image: nginx:latest
+    ports:
+      - 80:8080
+    volumes:
+      - ./nginx/nginx-proxy.conf:/etc/nginx/conf.d/default.conf:ro
+      - ./web/build:/var/www/frontend # maps frontend build inside nginx
```

Comment:
hmm, in dev mode we should be using webpack dev server? that would mean we just need nginx to route to that server's port. running webpack in docker is known problem, large amount of file IO is not docker+non-linux strongpoint. i usually end up using cross-env to support running simple shell commands on windows and mac to run webpack in host. 

dont get me wrong, docker is important for repeatable builds, but in prod, the assets would be compiled using multi-stage docker build, and uploaded into nginx/cdn folder

Reply:
Yeah I wasn't able to really figure out setting up hot-reloading and whatnot for dev mode. Could use some help on this part 🐙